daemon/cmd/daemon_main.go

```diff
@@ -815,6 +815,9 @@ func init() {
 	flags.Int(option.FragmentsMapEntriesName, defaults.FragmentsMapEntries, "Maximum number of entries in fragments tracking map")
 	option.BindEnv(option.FragmentsMapEntriesName)
 
+	flags.Int(option.LBMapEntriesName, defaults.LBMapEntries, "Maximum number of entries in Cilium BPF lbmap")
```

Comment:
I'd keep the default in `pkg/lbmap` package.

Reply:
I implemented it similar to how we populate `option.Config.PolicyMapEntries`. We use default value of PolicyMapEntries from `pkg/defaults`(https://github.com/cilium/cilium/blob/master/daemon/cmd/daemon_main.go#L714-L715). Should I change that as well, so that we maintain consistency?

Reply:
> Should I change that as well, so that we maintain consistency?

If @cilium/policy is OK with it, then yes please.